test_webhook.py

```
@@ -0,0 +1,80 @@
+"""
+
+import requests
+import json
+import time
+
+def test_webhook_endpoints():
+    base_url = "http://localhost:8000"
+    
+    # Test health endpoint
+    try:
+        response = requests.get(f"{base_url}/health")
```

Ruff (0.12.2)

Probable use of `requests` call without timeout

(S113)